certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -179,70 +153,6 @@ def _prepare_pebble_server(self) -> None:
 
         print('=> Finished pebble instance deployment.')
 
-    def _prepare_boulder_server(self) -> None:
-        """Configure and launch the Boulder server"""
-        print('=> Starting boulder instance deployment...')
-        instance_path = join(self._workspace, 'boulder')
-
-        # Load Boulder from git, that includes a docker-compose.yml ready for production.
-        process = self._launch_process(['git', 'clone', 'https://github.com/letsencrypt/boulder',
-                                        '--single-branch', '--depth=1', instance_path])
-        process.wait(MAX_SUBPROCESS_WAIT)
-
-        # Allow Boulder to ignore usual limit rate policies, useful for tests.
-        ref = importlib_resources.files("certbot_integration_tests")
-        ref = ref / "assets" / "boulder-rate-limit-policies.yml"
-        with importlib_resources.as_file(ref) as path:
-            shutil.copyfile(path, join(instance_path, 'test/rate-limit-policies.yml'))
-
-        if self._dns_server:
-            # Change Boulder config to use the provided DNS server
-            for suffix in ["", "-remote-a", "-remote-b"]:
-                with open(join(instance_path, 'test/config/va{}.json'.format(suffix)), 'r') as f:
-                    config = json.loads(f.read())
-                config['va']['dnsResolvers'] = [self._dns_server]
-                with open(join(instance_path, 'test/config/va{}.json'.format(suffix)), 'w') as f:
-                    f.write(json.dumps(config, indent=2, separators=(',', ': ')))
-
-        # This command needs to be run before we try and terminate running processes because
-        # docker compose up doesn't always respond to SIGTERM. See
-        # https://github.com/certbot/certbot/pull/9435.
-        self._register_preterminate_cmd(['docker', 'compose', 'down'], cwd=instance_path)
```

Comment:
i like deleting code. do you like deleting code?

if you do, i think that with this change, `self._preterminate_cmds_args`, all uses of it, and functions related to it can be deleted. totally up to you if you want to do this now/ever though

Reply:
a good PR is one with an extremely high deleted-to-added line ratio, and if i can make mine higher, i'm on it!